contract-examples/test/contract_deployer_allow_list.ts

```diff
@@ -1,191 +1,115 @@
 // (c) 2019-2022, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
-import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
-import { expect } from "chai";
-import {
-  Contract,
-  ContractFactory,
-} from "ethers"
 import { ethers } from "hardhat"
+const assert = require('assert')
 
 // make sure this is always an admin for minter precompile
-const adminAddress: string = "0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"
-const ALLOWLIST_ADDRESS = "0x0200000000000000000000000000000000000000";
+const ADMIN_ADDRESS: string = "0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"
+const DEPLOYER_ALLOWLIST_ADDRESS = "0x0200000000000000000000000000000000000000";
 
-const ROLES = {
-  NONE: 0,
-  DEPLOYER: 1,
-  ADMIN: 2
-};
+const testFn = (fnNameOrNames: string | string[], overrides = {}, debug = false) => async function () {
```

Comment:
I'd slightly prefer `test_blahBlahBlah` since it avoids building function names with strings. Could go either way so long as the `test` fn has a comment explaining what it does such as the one I mentioned.

Reply:
I actually prefer `test_blahBlah` as well, it's a little (not a lot) more obvious how it maps to the smart contract call. I changed this already